scala_proto/scala_proto.bzl

```diff
@@ -316,21 +316,18 @@ def _colon_paths(data):
 def _gen_proto_srcjar_impl(ctx):
     acc_imports = depset()
 
-    proto_deps, java_proto_lib_deps = [], []
+    proto_deps, jvm_deps = [], []
     for target in ctx.attr.deps:
         if hasattr(target, 'proto'):
             proto_deps.append(target)
             acc_imports += target.proto.transitive_sources
         else:
-            java_proto_lib_deps.append(target)
+            jvm_deps.append(target)
 
-    if not ctx.attr.with_java and len(java_proto_lib_deps) > 0:
```

Comment:
this removal is related to #315 ?

Reply:
No I hadn't seen that issue. I'll need to investigate that too

Reply:
so why did you remove it? not saying it's wrong I just don't understand how it's related to the current change

Reply:
Because I want to allow for scala_library deps even if with_java is false

Reply:
oh I see. thanks